mithril-signer/src/cardano_transactions_preloader_activation_according_to_aggregator.rs

```diff
@@ -0,0 +1,120 @@
+use std::sync::Arc;
+
+use anyhow::Context;
+use async_trait::async_trait;
+
+use mithril_common::{
+    cardano_transactions_preloader::CardanoTransactionsPreloaderActivationState,
+    entities::SignedEntityTypeDiscriminants, StdResult,
+};
+
+use crate::AggregatorClient;
+
+/// CardanoTransactionsPreloaderActivationAccordingToAggregator
+pub struct CardanoTransactionsPreloaderActivationAccordingToAggregator {
+    aggregator_client: Arc<dyn AggregatorClient>,
+}
+
+impl CardanoTransactionsPreloaderActivationAccordingToAggregator {
```

Comment:
The name is a bit cumbersome:
```suggestion
impl CardanoTransactionsPreloaderActivationSigner {
```

So is the file name, maybe this is enough `cardano_transactions_preloader_checker.rs`.

Reply:
Done ✅